src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de.json

```diff
@@ -172,7 +172,7 @@
         "titlePageSettings": "Einstellungen",
         "headerPageSettings": "Basis-Informationen",
         "titleBlockOverview": "Blöcke",
-        "titleBlockSettings": "Block-Einstellungen",
+        "titleBlockSettings": "Block-Einstellungen ({type})",
```

Comment:
Please do not do this as a snippet solution. The translations themselves are fine!